.eslintrc.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  extends: ['eslint:recommended', 'google'],
+  env: {
+    serviceworker: true,
+    browser: true,
+    node: true,
+    es6: true,
+  },
+  parserOptions: {
+    ecmaVersion: 2017,
+    sourceType: 'module',
+  },
+  globals: {
+    BroadcastChannel: false,
+    WorkboxSW: false,
+    workbox: false,
+  },
+  rules: {
+    'no-tabs': 2,
+    'linebreak-style': 0,
+    'space-before-function-paren': ['error', {
```

Comment:
I've also updated the  `eslint-config-google` rule for this, as I think it's what the Google JS style guide recommends. You can get the changes in `0.9.0`.

Also, in the future, if you're tempted to add rules/exceptions to your eslint config, feel free to ping me, as maybe it's a mistake :)